.github/workflows/codecov.yml

```diff
@@ -0,0 +1,22 @@
+name: "Codecov"
+
+on: [push, pull_request]
```

Comment:
Why on push? I feel like it maybe should be a daily activity more so than after every push.

Reply:
If we want to leverage the Codecov comments feature of PRs, then each PR needs to be compared to the header commit. In this scenario, opting for code review on push might be more suitable.

Do you think the PR comment feature is necessary?

Reply:
That makes sense. Would you mind putting a comment explaining that?

Reply:
I'd be happy to.